docs/csharp/codedoc.md

```diff
@@ -169,7 +183,7 @@ public class Math
 
 ### &lt;returns&gt;
 
-As the name suggests you use the `<returns>` tag in the comment for a method declaration to describe its return value.
+The `<returns>` tag describes the return value of a method declaration.
 Like before this will be illustrated on the first `Add` method go ahead an implement it on other methods.
```

Comment:
missing comma after before, 

the second part of the sentence needs work. It should probably be rewritten to something like:
`Add` method. You can do the same on other methods.